### PR TITLE
langchain: Update `UpstashRedisEntityStore` class properties

### LINKED ISSUE #18891: `UpstashRedisEntityStore` initializer does not work. Upstash Redis Instance could not be created.
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

The following code will not work:

```
from langchain.memory.entity import UpstashRedisEntityStore

entity_store = UpstashRedisEntityStore(
    session_id="my-session",
    url="your-upstash-url",
    token="your-upstash-redis-token",
    ttl=600,
)
```

### Error Message and Stack Trace (if applicable)

Upstash Redis instance could not be initiated.
Traceback (most recent call last):
  File "/Users/albertpurnama/Documents/dev/qornetto/main.py", line 27, in <module>
    entity_store = UpstashRedisEntityStore(
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/albertpurnama/Documents/dev/qornetto/.venv/lib/python3.11/site-packages/langchain/memory/entity.py", line 106, in __init__
    self.session_id = session_id
    ^^^^^^^^^^^^^^^
  File "/Users/albertpurnama/Documents/dev/qornetto/.venv/lib/python3.11/site-packages/pydantic/v1/main.py", line 357, in __setattr__
    raise ValueError(f'"{self.__class__.__name__}" object has no field "{name}"')
ValueError: "UpstashRedisEntityStore" object has no field "session_id"

### Description

I'm trying to use `UpstashRedisEntityStore` but the initializer does not work.

### System Info

langchain==0.1.11
langchain-community==0.0.27
langchain-core==0.1.30
langchain-openai==0.0.8
langchain-text-splitters==0.0.1
langchainhub==0.1.15

mac

python 3.11.6

### PR DESCRIPTION
**Pull request by albertpurnama**
Thank you for contributing to LangChain!

- [x] **PR title**: "package: description"
  - Where "package" is whichever of langchain, community, core, experimental, etc. is being modified. Use "docs: ..." for purely docs changes, "templates: ..." for template changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"

- [x] **PR message**:
**Description:** We're adding some missing properties from `UpstashRedisEntityStore` class.
**Issue:** resolves #18891 
**Dependencies:** None
**Twitter handle:** https://twitter.com/albertpurnama

- [x] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, hwchase17.

### CODE REVIEW

**Comment by baskaryan** [comment]
what's the motivation for the change?

**Comment by albertpurnama** [comment]
Thanks for looking over this issue @baskaryan, I'm using it currently in my local project.

I opened a more detailed explanation on the error message in the description's linked [issue](https://github.com/langchain-ai/langchain/issues/18891) as well as a minimum reproducible code. Trace log is also included. How can I improve the details on this issue? Or am I misunderstanding your question?

**Comment by baskaryan** [comment]
> Thanks for looking over this issue @baskaryan, I'm using it currently in my local project.
> 
> I opened a more detailed explanation on the error message in the description's linked [issue](https://github.com/langchain-ai/langchain/issues/18891) as well as a minimum reproducible code. Trace log is also included. How can I improve the details on this issue? Or am I misunderstanding your question?

missed the linked issue, that's perfect thanks!

File: libs/langchain/langchain/memory/entity.py
Comment (baskaryan) [inline libs/langchain/langchain/memory/entity.py:1]: if init vars are now fields on the pydantic model believe the proper way to initialize (so pydantic validation occurs) is to pass them to super().init
```python
    def __init__(
        self,
        session_id: str = "default",
        url: str = "",
        token: str = "",
        key_prefix: str = "memory_store",
        ttl: Optional[int] = 60 * 60 * 24,
        recall_ttl: Optional[int] = 60 * 60 * 24 * 3,
        *args: Any,  # shouldn't be here, pydantic models don't accepts args
        **kwargs: Any,
    ):
        if args:
          raise ValueError(f"Received unsupported positional args {args}")

        try:
            from upstash_redis import Redis
        except ImportError:
            raise ImportError(
                "Could not import upstash_redis python package. "
                "Please install it with `pip install upstash_redis`."
            )
        super().__init__(
          redis_client = Redis(url=url, token=token), # shouldn't try/except this if it's expected downstream
          session_id=session_id, 
          key_prefix=key_prefix,
          ttl=ttl,
          recall_ttl=recall_ttl or ttl,
          **kwargs
        )
        
```

**Comment by ccurme** [comment]
Closing as author is unresponsive, can re-open if needed.

**Pull request closed without merging**